### PR TITLE
Explain our workflow in AWS further

### PR DESCRIPTION
**Pull request by ezr-ondrej**
This tries to improve our AWS guide to explain all the quirks. It tries to explain the production workflow, while also give away easy steps for development.

### CODE REVIEW

File: docs/configure-amazon-role.md [hunk 1]
```diff
@@ -1,29 +1,38 @@
...
+This is the account that this application needs the credentials and will use to connect to AWS services.
 
-* Navigate to Identity and Access Management (IAM) on AWS.
-* Click on Policies - Create policy.
-* Click on JSON tab and paste the code below.
-* Click on your login name in the top-right corner and copy the Account ID to the clipboard. Then edit the "Resource" string and replace `123456789` with your account ID.
-* Click Next, Next.
-* Give the policy a name: `redhat-provisioning-service`
-* Click on Create Policy
-* Select **Access key - Programmatic access** for access type.
```
Comment (ezr-ondrej) [inline docs/configure-amazon-role.md:25]: removing this, I believe it is not relevant to Policy, right?

Comment (lzap) [reply docs/configure-amazon-role.md:25]: Perhaps copy & paste error.

File: docs/configure-amazon-role.md [hunk 2]
```diff
@@ -113,16 +127,26 @@ There can be multiple tenant accounts defined, therefore it is good to give them
...
 
-### Configuring on prod/stage
+### Configuring on ConsoleDot prod/stage
 
 Go to Sources App, create new Provisioning Source, select Manually and enter the AWS ARN string. _This needs to be elaborated more, we do not have the feature fully implemented._
 
 ## Configuring AWS CloudWatch
 
+*This is bit unrelated, but It is another use of AWS services by our app, so it's together for the time being.*
+
```
Comment (lzap) [inline docs/configure-amazon-role.md:149]: Feel free to simple move it out into a separate file.

**Review by lzap — APPROVED** [review]
I haven't checked creating this step-by-step, but I don't see any problems.

**Comment by ezr-ondrej** [comment]
Merging, we can follow-up with further improvements :)

**Pull request merged**